### PR TITLE
Bug 1847450: Publish only one console link for Kibana

### PR DESCRIPTION
**Pull request by periklis**
To address: https://bugzilla.redhat.com/show_bug.cgi?id=1847450

**Needs backport:** release-4.5

cc @jhadvig

### CODE REVIEW

File: pkg/k8shandler/kibana/kibana_test.go [hunk 1]
```diff
@@ -158,19 +140,12 @@ var _ = Describe("Reconciling", func() {
 			It("should create two new console links for the Kibana route", func() {
```
Comment (jcantrill) [inline pkg/k8shandler/kibana/kibana_test.go:140]: nit.. it should only create one

File: pkg/k8shandler/kibana/kibana_test.go [hunk 2]
```diff
@@ -218,27 +193,20 @@ var _ = Describe("Reconciling", func() {
...
 					sharingConfigReader,
 					sharingConfigReaderBinding,
-					consoleAppLogsLink,
-					consoleInfraLogsLink,
+					consoleLink,
 				)
 				esClient = newFakeEsClient(client, fakeResponses)
 			})
 
 			It("should replace existing sharing confimap links with two console links", func() {
```
Comment (jcantrill) [inline pkg/k8shandler/kibana/kibana_test.go:201]: same here

File: pkg/k8shandler/kibana/route.go [hunk 3]
```diff
@@ -133,26 +130,19 @@ func (clusterRequest *KibanaRequest) createOrUpdateKibanaRoute() error {
 	return nil
 }
 
-func (clusterRequest *KibanaRequest) createOrUpdateKibanaConsoleLinks() error {
+func (clusterRequest *KibanaRequest) createOrUpdateKibanaConsoleLink() error {
```
Comment (jcantrill) [inline pkg/k8shandler/kibana/route.go:133]: can we take the opportunity to rename this to somethign like "ReconcileXXXXX"

Comment (periklis) [reply pkg/k8shandler/kibana/route.go:133]: Let's not start renaming here. The whole reconciler is full of `createOrUpdate..`.

**Pull request merged**